ONSdigital/eq-survey-runner
# Renames "Response" to "Answer" as agreed by team

### What is the context of this PR?

Following our software design review the team felt there were a number of changes that needed to be made to the existing codebase to make it clearer to understand, and better support the features that we have planned.  This pull request is the first of a series of pull requests which address the changes we discussed and agreed.

Specifically, this PR changes the name of the "Response" object used in our data model to "Answer" in the hope of clearing up any confusion created when discussing responses.
### How to review

1) Check out this branch
2) Run the tests
3) Run the app and verify that it continues to function as before
### Who worked on the PR

@weapdiv-david worked on this PR, so it should be reviewed by all other members of the team.
